src/PIL/DdsImagePlugin.py

```diff
@@ -263,62 +264,43 @@ class D3DFMT(IntEnum):
 
 
 # Backward compat layer
-DDSD_CAPS = DDSD.CAPS
-DDSD_HEIGHT = DDSD.HEIGHT
-DDSD_WIDTH = DDSD.WIDTH
-DDSD_PITCH = DDSD.PITCH
-DDSD_PIXELFORMAT = DDSD.PIXELFORMAT
-DDSD_MIPMAPCOUNT = DDSD.MIPMAPCOUNT
-DDSD_LINEARSIZE = DDSD.LINEARSIZE
-DDSD_DEPTH = DDSD.DEPTH
-
-DDSCAPS_COMPLEX = DDSCAPS.COMPLEX
-DDSCAPS_TEXTURE = DDSCAPS.TEXTURE
-DDSCAPS_MIPMAP = DDSCAPS.MIPMAP
-
-DDSCAPS2_CUBEMAP = DDSCAPS2.CUBEMAP
-DDSCAPS2_CUBEMAP_POSITIVEX = DDSCAPS2.CUBEMAP_POSITIVEX
-DDSCAPS2_CUBEMAP_NEGATIVEX = DDSCAPS2.CUBEMAP_NEGATIVEX
-DDSCAPS2_CUBEMAP_POSITIVEY = DDSCAPS2.CUBEMAP_POSITIVEY
-DDSCAPS2_CUBEMAP_NEGATIVEY = DDSCAPS2.CUBEMAP_NEGATIVEY
-DDSCAPS2_CUBEMAP_POSITIVEZ = DDSCAPS2.CUBEMAP_POSITIVEZ
-DDSCAPS2_CUBEMAP_NEGATIVEZ = DDSCAPS2.CUBEMAP_NEGATIVEZ
-DDSCAPS2_VOLUME = DDSCAPS2.VOLUME
-
-DDPF_ALPHAPIXELS = DDPF.ALPHAPIXELS
-DDPF_ALPHA = DDPF.ALPHA
-DDPF_FOURCC = DDPF.FOURCC
-DDPF_PALETTEINDEXED8 = DDPF.PALETTEINDEXED8
-DDPF_RGB = DDPF.RGB
-DDPF_LUMINANCE = DDPF.LUMINANCE
-
-DDS_FOURCC = DDPF_FOURCC
-DDS_RGB = DDPF_RGB
-DDS_RGBA = DDPF_RGB | DDPF_ALPHAPIXELS
-DDS_LUMINANCE = DDPF_LUMINANCE
-DDS_LUMINANCEA = DDPF_LUMINANCE | DDPF_ALPHAPIXELS
-DDS_ALPHA = DDPF_ALPHA
-DDS_PAL8 = DDPF_PALETTEINDEXED8
-
-DDS_HEADER_FLAGS_TEXTURE = DDSD_CAPS | DDSD_HEIGHT | DDSD_WIDTH | DDSD_PIXELFORMAT
-DDS_HEADER_FLAGS_MIPMAP = DDSD_MIPMAPCOUNT
-DDS_HEADER_FLAGS_VOLUME = DDSD_DEPTH
-DDS_HEADER_FLAGS_PITCH = DDSD_PITCH
-DDS_HEADER_FLAGS_LINEARSIZE = DDSD_LINEARSIZE
-
-DDS_HEIGHT = DDSD_HEIGHT
-DDS_WIDTH = DDSD_WIDTH
-
-DDS_SURFACE_FLAGS_TEXTURE = DDSCAPS_TEXTURE
-DDS_SURFACE_FLAGS_MIPMAP = DDSCAPS_COMPLEX | DDSCAPS_MIPMAP
-DDS_SURFACE_FLAGS_CUBEMAP = DDSCAPS_COMPLEX
-
-DDS_CUBEMAP_POSITIVEX = DDSCAPS2_CUBEMAP | DDSCAPS2_CUBEMAP_POSITIVEX
-DDS_CUBEMAP_NEGATIVEX = DDSCAPS2_CUBEMAP | DDSCAPS2_CUBEMAP_NEGATIVEX
-DDS_CUBEMAP_POSITIVEY = DDSCAPS2_CUBEMAP | DDSCAPS2_CUBEMAP_POSITIVEY
-DDS_CUBEMAP_NEGATIVEY = DDSCAPS2_CUBEMAP | DDSCAPS2_CUBEMAP_NEGATIVEY
-DDS_CUBEMAP_POSITIVEZ = DDSCAPS2_CUBEMAP | DDSCAPS2_CUBEMAP_POSITIVEZ
-DDS_CUBEMAP_NEGATIVEZ = DDSCAPS2_CUBEMAP | DDSCAPS2_CUBEMAP_NEGATIVEZ
+module = sys.modules[__name__]
+for item in DDSD:
+    setattr(module, "DDSD_" + item.name, item.value)
+for item in DDSCAPS:
+    setattr(module, "DDSCAPS_" + item.name, item.value)
+for item in DDSCAPS2:
+    setattr(module, "DDSCAPS2_" + item.name, item.value)
+for item in DDPF:
+    setattr(module, "DDPF_" + item.name, item.value)
```

Comment:
That's a great idea, but it won't work for code auto-completion, but since these are old/obsolete constants(they are replaced by enums) it's all good